### DISCUSSION TITLE
请问在archlinux下要如何使用

### DISCUSSION THREAD

**Discussion by ZavierYan** [BeyondDimension/SteamTools#888] [Q&A] [answered]
archlinux下不能用包管理器安装，我直接解包7z文件到用户主目录后使用不能正常启动。
我不太确定是不是我的操作有问题，希望各位大佬解惑。

**Comment by Mossimos** [comment, marked as answer]
解压后直接启动这个 Steam++ 或者执行 Steam++.sh 里面的命令启动

![image](https://user-images.githubusercontent.com/26396340/149288442-8b241e52-f115-4bc0-b674-a19e4385501a.png)
![image](https://user-images.githubusercontent.com/26396340/149288649-4521082b-32ad-4af6-819a-9440c9a95d2f.png)
